common/dna_data/wc_orc_dna.txt

```diff
@@ -0,0 +1,1195 @@
+ 		gene_bs_nose_ridge_angle={ "nose_ridge_angle_neg" 133 "nose_ridge_angle_pos" 22 }
+ 		gene_bs_nose_ridge_width={ "nose_ridge_width_pos" 255 "nose_ridge_width_pos" 165 }
+ 		gene_bs_nose_size={ "nose_size_neg" 179 "nose_size_neg" 11 }
+ 		gene_bs_nose_tip_angle={ "nose_tip_angle_pos" 59 "nose_tip_angle_pos" 176 }
+ 		gene_bs_nose_tip_forward={ "nose_tip_forward_pos" 55 "nose_tip_forward_neg" 177 }
+ 		gene_bs_nose_tip_width={ "nose_tip_width_neg" 79 "nose_tip_width_neg" 122 }
+ 		face_detail_cheek_def={ "cheek_def_02" 82 "cheek_def_01" 147 }
+ 		face_detail_cheek_fat={ "cheek_fat_02_pos" 200 "cheek_fat_04_pos" 238 }
+ 		face_detail_chin_cleft={ "chin_cleft" 14 "chin_cleft" 14 }
+ 		face_detail_chin_def={ "chin_def" 32 "chin_def" 94 }
+ 		face_detail_eye_lower_lid_def={ "eye_lower_lid_def" 255 "eye_lower_lid_def" 252 }
+ 		face_detail_eye_socket={ "eye_socket_01" 160 "eye_socket_02" 65 }
+ 		face_detail_nasolabial={ "nasolabial_01" 237 "nasolabial_01" 229 }
+ 		face_detail_nose_ridge_def={ "nose_ridge_def_neg" 0 "nose_ridge_def_neg" 12 }
+ 		face_detail_nose_tip_def={ "nose_tip_def" 0 "nose_tip_def" 23 }
+ 		face_detail_temple_def={ "temple_def" 0 "temple_def" 229 }
+ 		expression_brow_wrinkles={ "brow_wrinkles_04" 255 "brow_wrinkles_03" 202 }
+ 		expression_eye_wrinkles={ "eye_wrinkles_03" 193 "eye_wrinkles_01" 193 }
+ 		expression_forehead_wrinkles={ "forehead_wrinkles_03" 255 "forehead_wrinkles_01" 255 }
+ 		expression_other={ "cheek_wrinkles_both_01" 0 "cheek_wrinkles_both_01" 209 }
+ 		complexion={ "complexion_7" 185 "complexion_2" 88 }
+ 		gene_height={ "normal_height" 255 "normal_height" 141 }
+ 		gene_bs_body_type={ "body_fat_head_fat_low" 97 "body_fat_head_fat_low" 171 }
+ 		gene_bs_body_shape={ "body_shape_average_clothed" 255 "body_shape_pear_half" 233 }
+ 		gene_bs_bust={ "bust_clothes" 8 "bust_shape_1_half" 171 }
+ 		gene_age={ "old_1" 218 "old_4" 249 }
+ 		gene_eyebrows_shape={ "avg_spacing_lower_thickness" 255 "avg_spacing_low_thickness" 125 }
+ 		gene_eyebrows_fullness={ "layer_2_low_thickness" 249 "layer_2_low_thickness" 160 }
+ 		gene_body_hair={ "body_hair_avg" 76 "body_hair_sparse" 138 }
+ 		hairstyles={ "northern_hairstyles" 191 "orcish_hairstyles" 21 }
+ 		beards={ "northern_beards" 195 "orcish_beards" 10 }
+ 		eye_accessory={ "normal_eyes" 219 "normal_eyes" 219 }
+ 		teeth_accessory={ "normal_teeth" 0 "normal_teeth" 0 }
+ 		eyelashes_accessory={ "no_eyelashes" 144 "no_eyelashes" 144 }
+ 		gene_bs_ear_lenght={ "erect_ear_lenght" 36 "erect_ear_lenght" 36 }
+ 		special_eyebrows={ "no_eyebrows" 255 "no_eyebrows" 13 }
+ 		special_eyes={ "no_special_eyes" 0 "no_special_eyes" 0 }
+ 		gene_bs_mouth_tusks={ "mouth_tusks_pos" 255 "mouth_tusks_pos" 255 }
+ 		static_tusks={ "orcish_tusks" 237 "orcish_tusks" 237 }
+ 		gene_bs_ear_tip={ "ear_tip_pos" 255 "ear_tip_pos" 255 }
+            gene_skin_value={ "skin_value_neg" 0 "skin_value_neg" 0 }
+		}
+	}
+	enabled=yes
+}
+
+thrall_dna_2 = { # base by Garthor13, adjusted by ercarp
```

Comment:
He lacks hair and beards.
<details>
<summary>Click to expand</summary>

![Uploading image.png…]()

</details>

Reply:
Fixed in ed5cd8f2efd73481cb3dcf379af77f208a2ef5df.